src/main/scala/firrtl/Utils.scala

```diff
@@ -114,22 +114,22 @@ object Utils extends LazyLogging {
      t match {
        case (_: GroundType) => f
        case (t: BundleType) => 
-         val (_, flip) = ((t.fields foldLeft (i, None: Option[Orientation])){
-            case ((n, ret), x) if n < get_size(x.tpe) => ret match {
-              case None => (n, Some(get_flip(x.tpe,n,times(x.flip,f))))
-              case Some(_) => (n, ret)
-            }
-            case ((n, ret), x) => (n - get_size(x.tpe), ret)
-         })
+         val (_, flip) = (t.fields foldLeft(i, None: Option[Orientation])) {
```

Comment:
folds aren't arity-1 and should be more explicitly done as a function call.

(After seeing this, I realize that I misspoke earlier when I referred to fold as a common enough function to have this syntax.)